### ISSUE TITLE
Scenes

### ISSUE THREAD

**Issue by ejjenkins** [adityapattiyeri/home-remote#2]
I am in the process of recreating this project.  I have a pretty handle of the entire thing, but I can't seem to get what the scenes portion is doing.  Would you be willing to explain this for me.

I know this is not an actual issue (well, it is for me), but it the only way to ask about this.

Thank you!

**Comment by adityapattiyeri** [comment]
Sure, I would love to help you out!!

 The scenes feature actually uses an input select in home assistant. It contains all the values like morning, evening , night, etc. The idea being, the home can only be in state at a time. I then use the value of the input select within automations to actually change the states of the devices or "change the scene". It is a little bit confusing because I'm not actually using the "scenes" feature of home assistant. You can find the screenshot of my input select helper below. Do let me know if you need more help and I would love to see your implementation!

<img width="380" alt="Screenshot 2022-08-21 at 12 52 05 PM" src="https://user-images.githubusercontent.com/61015809/185780441-dcdd2812-fda4-45e9-bcd9-daf9317a2eec.png">

**Comment by ejjenkins** [comment]
I appreciate the response.  I have just never seen the input_select piece.  I read the Home Assistant docs on input_select and I will play around and make sure I get it.  Thank you!

Here is what I have done so far:
I am using an ILI9341 2.4" LCD for the display and a dual 18650 battery board/charger for power.  I will design and 3D print a case for it.  Here are pics of the Home page and Lights page.  I am adding a second Lights page.  On the Home page I added 6 sensors that simply show the state...and on each lights page I will have 11 devices.
For now I have added the button, like yours, but I will come up with a permanent solution once I start drawing up the printed case.

This is a great project, I didn't know I needed this until I saw it.

![20220820_121633](https://user-images.githubusercontent.com/14811415/185795501-3de10c42-4e3d-4c4d-8dcc-8afbcaca7402.jpg)
![20220820_121648](https://user-images.githubusercontent.com/14811415/185795508-72a6a92a-549c-44bc-8dc2-fca15e765aa1.jpg)
![20220821_102120](https://user-images.githubusercontent.com/14811415/185795604-a911b5ed-66c0-4f14-aca6-67a858850fc3.jpg)
